<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AsiakasToimittajaDlg</class>
 <widget class="QDialog" name="AsiakasToimittajaDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asiakkaan / toimittajan tiedot</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nimi</string>
     </property>
     <property name="buddy">
      <cstring>nimiEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nimiEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Maa</string>
     </property>
     <property name="buddy">
      <cstring>maaCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="maaCombo"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="yLabel">
     <property name="text">
      <string>Y-tunnus</string>
     </property>
     <property name="buddy">
      <cstring>yEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="yEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="alvlabel">
     <property name="text">
      <string>Alv-tunnus</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="alvEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Sähköposti</string>
     </property>
     <property name="buddy">
      <cstring>emailEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="emailEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Lähiosoite</string>
     </property>
     <property name="buddy">
      <cstring>osoiteEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPlainTextEdit" name="osoiteEdit">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Postiosoite</string>
     </property>
     <property name="buddy">
      <cstring>postinumeroEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="postinumeroEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="kaupunkiEdit"/>
     </item>
    </layout>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="tilitLabel">
     <property name="text">
      <string>Pankkitilit</string>
     </property>
     <property name="buddy">
      <cstring>tilitLista</cstring>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QListWidget" name="tilitLista">
     <property name="editTriggers">
      <set>QAbstractItemView::AllEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="ovtLabel">
     <property name="text">
      <string>Verkkolaskuosoite</string>
     </property>
     <property name="buddy">
      <cstring>ovtEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="ovtEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="valittajaLabel">
     <property name="text">
      <string>Välittäjätunnus</string>
     </property>
     <property name="buddy">
      <cstring>valittajaEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="valittajaEdit"/>
   </item>
   <item row="11" column="1">
    <widget class="CheckCombo" name="ryhmaCombo"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="ryhmatLabel">
     <property name="text">
      <string>Ryhmät</string>
     </property>
     <property name="buddy">
      <cstring>ryhmaCombo</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckCombo</class>
   <extends>QComboBox</extends>
   <header location="global">tools/checkcombo.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nimiEdit</tabstop>
  <tabstop>maaCombo</tabstop>
  <tabstop>yEdit</tabstop>
  <tabstop>alvEdit</tabstop>
  <tabstop>emailEdit</tabstop>
  <tabstop>osoiteEdit</tabstop>
  <tabstop>postinumeroEdit</tabstop>
  <tabstop>kaupunkiEdit</tabstop>
  <tabstop>valittajaEdit</tabstop>
  <tabstop>ovtEdit</tabstop>
  <tabstop>tilitLista</tabstop>
  <tabstop>ryhmaCombo</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AsiakasToimittajaDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>615</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AsiakasToimittajaDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>615</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
